src/overlay_29_02338604.c

```diff
@@ -0,0 +1,14 @@
+#include "overlay_29_02338604.h"
+
+#include "dungeon.h"
+
+extern struct dungeon* DUNGEON_PTR[];
+
+void SetShouldBoostHiddenStairsSpawnChance(u8 should_boost)
+{
+    #ifdef JAPAN
+    DUNGEON_PTR[0]->unknown_file_buffer_0x12162[0x91f]=should_boost;
```

Comment:
This is likely a variable for boosting the hidden stairs chance that is not yet documented in the dungeon struct. It seems unlikely that this info would be stored in a file buffer, or within an array for that matter. Perhaps we can shorten the file buffer, add the new variable, and add another filler array to ensure that the dungeon struct remains the same size.

Reply:
Looks like I did my math wrong and overran the end of this array- it's actually the later (appropriately-named) variable boost_hidden_stairs_spawn_chance. Fixed.